COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
> I can extract the names of all committers from the git log and add them here so there won't be false exclusivity. How does that sound?

I think it would be best to do that (a) in a separate PR and (b) only after discussing this topic on the nix-dev mailing list. This is not something that we can decide easily.